app-modules/service-management/src/Models/ServiceRequest.php

```diff
@@ -218,6 +218,16 @@ public function histories(): HasMany
         return $this->hasMany(ServiceRequestHistory::class);
     }
 
+    public function feedback(): HasMany
+    {
+        return $this->hasMany(ServiceRequestFeedback::class);
+    }
+
+    public function latestFeedback(): HasOne
+    {
+        return $this->feedback()->one()->latestOfMany();
+    }
```

Comment:
Because we currently have no plans to allow multiple feedback items, can this be changed to a `hasOne` relation and get rid of `latestFeedback`?

If we do so we could easily change it back to a `hasMany` later, right?

Reply:
Hi,

Yep you're right ! we can change this later to HasMany again !

Thank you.